src/main/java/com/sap/cloud/yaas/service_sdk/example/java_jersey_wishlist/api/DefaultWishlistsResource.java

```diff
@@ -0,0 +1,109 @@
+
```

Comment:
Do we really need this new file? To me it looks similar to the existing [DefaultWishlistsResource.java](https://github.com/SAP/yaas_java_jersey_wishlist/blob/master/src/main/java/com/sap/wishlist/api/generated/DefaultWishlistsResource.java)